lib/uktt/geographical_area.rb

```diff
@@ -0,0 +1,41 @@
+module Uktt
+  class GeographicalArea
+    attr_accessor :config, :response
+
+    def initialize(opts = {})
+      Uktt.configure(opts)
+      @config = Uktt.config
+      @response = nil
+    end
+
+    def retrieve
+      fetch "#{GEOGRAPHICAL_AREAS}"
+    end
+
+    def config=(new_opts = {})
+      merged_opts = Uktt.config.merge(new_opts)
+      Uktt.configure merged_opts
```

Comment:
```suggestion
      Uktt.configure(merged_opts)
```

Reply:
Probably we could find and replace this I reckon